tensorflow/contrib/copy_graph/python/util/copy_elements.py

```diff
@@ -93,7 +93,7 @@ def copy_variable_to_graph(org_instance, to_graph, scope=''):
         trainable,
         name=new_name,
         collections=collections,
-        validate_shape=False)
+        validate_shape=True)
```

Comment:
Hi @alextp 
Sure I can add validate_shape as argument. But I argue that there's no reason to set the field validate_shape to False, since in order to create the new variable we need to evaluate the old variable to get the init_value, so we have the actual shape of variables. 
Do you have any examples in which the shape is not known at this point? Am I misunderstanding something?

Reply:
The problem with validate_shape is that it constrains the future shape of the variable. Some people create variables with validate_shape=False so they can assign values of different shapes to it.

Reply:
@alextp, Does really setting validate_shape=True constrains future shape of Variable?
If it does, any ref code for that?

Reply:
validate_shape=True means you cannot assign a value of a different shape to this variable

Reply:
Do you plan on updating this?

Reply:
Yes, I will post update my pull request in the following days. Will this package be moved in a different repository? I'm asking this since this script is inside `contrib` that will be deprecated in tf2.0.

Reply:
copy_graph as far as I know has no owner, so you shouldn't bother, unless you volunteer to take maintainership of it. @dynamicwebpaige for help about that